test/trento/hosts/host_test.exs

```diff
@@ -90,6 +90,7 @@ defmodule Trento.Hosts.HostTest do
           installation_source: installation_source,
           fully_qualified_domain_name: nil,
           prometheus_targets: prometheus_targets,
+          last_boot_timestamp: ~U[2024-01-01 00:00:00Z],
```

Comment:
improvement:
I think we could put this value in a variable like the rest of them. This way, we just need to give the value there.